content/en/contribute/docs/workflow.md

```diff
@@ -12,7 +12,7 @@ Anyone can contribute to CHT documentation by opening an issue in the [`cht-docs
 
 ### Basics
 
-* It is helpful to be comfortable with [git]({{< ref "core/guides/using-git" >}}) and [GitHub](https://lab.github.com/) to contribute to the CHT community.
+* It is helpful to be comfortable with [GitHub](https://lab.github.com/) to contribute to the CHT community.
```

Comment:
Instead of dropping the git link, how about just pointing to something more useful?

```suggestion
* It is helpful to be comfortable with [git](https://git-scm.com/doc/ext) and [GitHub](https://lab.github.com/) to contribute to the CHT community.
```

Reply:
ah - good call!